src/move/market_metadata/README.md

```diff
@@ -62,26 +62,55 @@ aptos move run \
 ```sh
 MARKET_METADATA=0xaaa...
 PROFILE=my-profile
-PROPERTY=string:foo
-VALUE=string:bar
-MARKET_ID=u64:1
+PROPERTY=foo
+VALUE=bar
+MARKET_ADDRESS=0xabcd
 ```
 
 <!-- markdownlint-disable MD013 -->
 
 ```sh
 aptos move run \
-    --args $MARKET_ID $PROPERTY $VALUE \
-    --function-id $MARKET_METADATA::emojicoin_dot_fun_market_metadata::add_market_property \
+    --args "address:$MARKET_ADDRESS" "string:[\"$PROPERTY\"]" "string:[\"$VALUE\"]" \
+    --function-id $MARKET_METADATA::emojicoin_dot_fun_market_metadata::add_market_properties \
     --profile $PROFILE
 ```
 
 ## Unset property
 
 ```sh
 aptos move run \
-    --args $MARKET_ID $PROPERTY \
-    --function-id $MARKET_METADATA::emojicoin_dot_fun_market_metadata::remove_market_property \
+    --args "address:$MARKET_ADDRESS" "string:[\"$PROPERTY\"]" \
+    --function-id $MARKET_METADATA::emojicoin_dot_fun_market_metadata::remove_market_properties \
```

Comment:
`remove_market_properties`? This looks like a copy-paste error

Did you test this?